src/lib/mocha/mocha-wrapper.js

```diff
@@ -10,41 +9,57 @@ var Mocha = require('mocha'),
 import {parseBoolean, parseNullableString, parseString} from '../environment-variable-parsers';
 import escapeRegExp from '../utils/escape-reg-exp';
 
-  var mochaConfig = JSON.parse(process.env.mochaConfig);
-mochaConfig.ui = 'fiberized-bdd-ui';
+class MochaWrapper {
+  constructor() {
+    let mochaConfig = JSON.parse(process.env.mochaConfig);
+    mochaConfig.ui = 'fiberized-bdd-ui';
 
-if (!mochaConfig.grep && parseBoolean(process.env['chimp.watch'])) {
-  mochaConfig.grep = new RegExp(parseString(process.env['chimp.watchTags']).split(',').map(escapeRegExp).join('|'));
-} else if (!mochaConfig.grep) {
-  mochaConfig.grep = new RegExp(
-    parseString(mochaConfig.tags).split(',').map(escapeRegExp).join('|')
-  );
-}
+    if (!mochaConfig.grep && parseBoolean(process.env['chimp.watch'])) {
+      mochaConfig.grep = new RegExp(parseString(process.env['chimp.watchTags']).split(',').map(escapeRegExp).join('|'));
+    } else if (!mochaConfig.grep) {
+      mochaConfig.grep = new RegExp(
+       parseString(mochaConfig.tags).split(',').map(escapeRegExp).join('|')
+      );
+    }
+
+    var mocha = new Mocha(mochaConfig);
+
+    mocha.addFile(path.join(path.resolve(__dirname, path.join('mocha-helper.js'))));
+    if (process.argv.length > 3) {
```

Comment:
this allows you to pass test files in a list.. test/test1.js test/test2.js test/test3.js . If you use circleci parallelization this is a must, since that's what circle provides to it's parallelized build machines

Looking at it now - I definitely agree, this could be made much more cleaner ;)